src/index.js

```diff
@@ -1,11 +1,24 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
+import './styles/index.css';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import index from './reducers/index';
+import App from './components/App';
+
+const initialState = [
+  { id: Math.floor(Math.random() * 100), title: 'Book1', category: 'Action' },
+  { id: Math.floor(Math.random() * 100), title: 'Book2', category: 'Biography' },
+  { id: Math.floor(Math.random() * 100), title: 'Book3', category: 'History' },
+];
+
+const store = createStore(index, { books: initialState });
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Provider store={store}>
+      <App />
+    </Provider>
```

Comment:
* [ ] It wraps `App` in `Provider` and assigns the whole thing into a variable (Currently, you are only missing to assign this to a variable and use it accordingly)

Reply:
I can't understand exactly what I must assign to a variable, I review other student's repos for this project and I can't see any difference or hint related to that.

Reply:
Hello Julian, this is what Microverse wants you to implement: 
![image](https://user-images.githubusercontent.com/38522578/132901714-510439ef-cc59-4690-b17b-5bd3028a136c.png)